IPython/terminal/ptshell.py

```diff
@@ -49,8 +49,7 @@ def get_completions(self, document, complete_event):
         )
         start_pos = -len(used)
         for m in matches:
-            yield Completion(m, start_position=start_pos)
-
+            yield Completion(m, start_position=start_pos, display_meta=m[::-1])
```

Comment:
Sweet ! Just for other information, the second column is the completion text reversed for now, as Jedi return a wrong value for now. We planned on adding extra info in this field later.